CMakeLists.txt

```diff
@@ -83,6 +83,13 @@ IF(NOT KOKKOS_HAS_TRILINOS)
   ENDif()
   IF(NOT DEFINED ${PROJECT_NAME})
```

Comment:
Shouldn't this be 
````
IF (NOT DEFINED PROJECT_NAME)
````
?

Reply:
I realize this line already exists, but I think this is wrong.  This line is 3 years old and somehow never go caught.

Reply:
This is obviously unrelated to the PR, per se, but should we fix it now?